app/scripts/lib/account-tracker.test.js

```diff
@@ -140,6 +143,24 @@ describe('Account Tracker', () => {
     });
   });
 
+  describe('onAccountRemoved', () => {
+    it('should call removeAccount method with the correct address when the `accountRemoved` event is emitted.', () => {
+      const mockAddress = '0x1234567890abcdef';
+      const removeAccountMock = jest.fn();
+      accountTracker.removeAccount = removeAccountMock;
+      mockEventEmitter.emit('accountRemoved', mockAddress);
+      expect(removeAccountMock).toHaveBeenCalledWith([mockAddress]);
+    });
+
+    it('should never call removeAccount method when `accountRemoved` is emitted with the wrong event name', () => {
+      const mockAddress = '0x1234567890abcdef';
+      const removeAccountMock = jest.fn();
+      accountTracker.removeAccount = removeAccountMock;
+      mockEventEmitter.emit('wrongAccountRemoved', mockAddress);
+      expect(removeAccountMock).not.toHaveBeenCalled();
+    });
+  });
```

Comment:
The same suggestions from `PreferencesController` should apply also for `AccountTracker`